site/src/pages/GroupsPage/SettingsGroupPage.tsx

```diff
@@ -35,7 +52,7 @@ export const SettingsGroupPage: FC = () => {
               add_users: [],
               remove_users: [],
             });
-            navigateToGroup();
+            navigate(`/groups/${data.name}`);
```

Comment:
oh, and small nit: I might set `{ replace: true }` here, since you're not really "navigating" to a new page, but rather the name of the current page is changing.